next.config.js

```diff
@@ -1,6 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
-}
+  transpilePackages: ["@jbrowse/react-circular-genome-view"],
```

Comment:
circular has to be transpiled if using nextjs

Reply:
why circular but not linear?

Reply:
circular & linear have different target ES versions. I believe circular is targeting es6+ which doesn't work in nextjs without transpiling.